scipy/stats/stats.py

```diff
@@ -5421,6 +5429,296 @@ def _betai(a, b, x):
 
 
 #####################################
+#       STATISTICAL DISTANCES       #
+#####################################
+
+def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
+    r"""
+    Compute the first Wasserstein distance between two 1D distributions.
+
+    The first Wasserstein distance between the distributions :math:`u` and
+    :math:`v` is:
+
+    .. math::
+
+        l_1 (u, v) = \inf_{\pi \in \Gamma (u, v)} \int_{\mathbb{R} \times
+        \mathbb{R}} |x-y| \mathrm{d} \pi (x, y)
```

Comment:
Rendered html docs look good:
https://3906-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.stats.wasserstein_distance.html#scipy.stats.wasserstein_distance
https://3906-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.stats.energy_distance.html#scipy.stats.energy_distance

However, our rule is that we normally try to only use LaTeX in the ``Notes`` section, because this isn't readable in a terminal (or Jupyter notebook or IDE help). So I suggest to move both this paragraph and the one on CDFs to the notes.

And same for ``energy_distance`` docstring